plugin/src/main/java/org/opensearch/ml/action/model_group/DeleteModelGroupTransportAction.java

```diff
@@ -93,6 +97,7 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Delete
 
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext()) {
             ActionListener<DeleteResponse> wrappedListener = ActionListener.runBefore(actionListener, context::restore);
+            // TODO: Remove this feature flag check once feature is GA, as it will be enabled by default
```

Comment:
Did you miss any code here, only comment. There is no flag.

Reply:
i moved the code but didn't remove comment. I'll remove it.